packages/location-field/src/index.js

```diff
@@ -0,0 +1,831 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+import {
+  Ban,
+  Briefcase,
+  Bus,
+  Home,
+  LocationArrow,
+  MapMarker,
+  MapPin,
+  Search,
+  Times
+} from "styled-icons/fa-solid";
+import { throttle } from "throttle-debounce";
+
+import {
+  currentPositionToLocation,
+  formatStoredPlaceName
+} from "@opentripplanner/core-utils/lib/map";
+import { isIE } from "@opentripplanner/core-utils/lib/ui";
+import getGeocoder from "@opentripplanner/geocoder";
+import { humainzeDistanceStringImperial } from "@opentripplanner/humanize-distance";
+import LocationIcon from "@opentripplanner/location-icon";
+
+import * as Styled from "./styled";
+
+// helper functions for dropdown options
```

Comment:
Since we're working in this micro-packages now, this might be a good opportunity to move these "helper functions" to a separate file in the package (similar to how the styled components are imported).